templates/stamp/stamp.json

```diff
@@ -0,0 +1,652 @@
+            "storageAccountName": "[variables('storageAccountName')]"
+        },
+        "jumpboxMachineSettings": {
+            "vmSize": "[parameters('jumpBoxVmSize')]",
+            "osImageReference": {
+                "publisher": "[variables('ubuntuOsConfigs').imagePublisher]",
+                "offer": "[variables('ubuntuOsConfigs').imageOffer]",
+                "sku": "[variables('ubuntuOsConfigs').imageSku]",
+                "version": "latest"
+            },
+            "dataDiskSize": 128
+        },
+        "mongoMachineSettings": {
+            "vmSize": "[parameters('mongoVmSize')]",
+            "osImageReference": {
+                "publisher": "[variables('ubuntuOsConfigs').imagePublisher]",
+                "offer": "[variables('ubuntuOsConfigs').imageOffer]",
+                "sku": "[variables('ubuntuOsConfigs').imageSku]",
+                "version": "latest"
+            },
+            "dataDiskSize": 256
+        },
+        "mysqlMachineSettings": {
+            "vmSize": "[parameters('mysqlVmSize')]",
+            "osImageReference": {
+                "publisher": "[variables('ubuntuOsConfigs').imagePublisher]",
+                "offer": "[variables('ubuntuOsConfigs').imageOffer]",
+                "sku": "[variables('ubuntuOsConfigs').imageSku]",
+                "version": "latest"
+            },
+            "dataDiskSize": 256
+        },
+        "frontendMachineSettings": {
+            "vmSize": "[parameters('frontendVmSize')]",
+            "osImageReference": {
+                "publisher": "[variables('ubuntuOsConfigs').imagePublisher]",
+                "offer": "[variables('ubuntuOsConfigs').imageOffer]",
+                "sku": "[variables('ubuntuOsConfigs').imageSku]",
+                "version": "latest"
+            },
+            "capacity": "[parameters('frontendVmSize')]",
+            "dataDiskSize": 256
+        },
+        "apiSettings": {
+            "apiVersion": "[variables('apiVersion')]",
+            "storageApiVersion": "[variables('storageApiVersion')]",
+            "virtualMachineApiVersion": "[variables('virtualMachineApiVersion')]",
+            "availabilitySetApiVersion": "[variables('availabilitySetApiVersion')]",
+            "virtualMachineExtensionsApiVersion": "[variables('virtualMachineExtensionsApiVersion')]",
+            "networkApiVersion": "[variables('networkApiVersion')]",
+            "computeApiVersion": "[variables('computeApiVersion')]",
+            "insightsApiVersion": "[variables('insightsApiVersion')]"
+        },
+
+        "mongodbHaInstallScriptBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/mongodb-high-availability/",
+        "azureSharedScriptUrl": "[concat('https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/shared_scripts/', variables('ubuntuOsConfigs').osName, '/')]",
+        "templateBaseUrl": "https://raw.githubusercontent.com/microsoft/oxa-tools/eltonc/templates/stamp/",

```

Comment:
templateBaseUrl - needs to be updated, and should probably be parameterized for branch (master/release/etc)